docs/metrics.md

```diff
@@ -124,39 +109,44 @@ Services.telemetry.scalarSet(
+
+Note that if the preceding two scalars are going to cause engineering headaches its possible to obtain equivalent values from the event telemetry (its just more work).
 
-- `datastoreCount` (integer). Current count of the number of items in the user's datastore. Note that this scalar is only updated when the user renders their full item list, either in the management view or in the doorhanger. So when testing whether this scalar is accurately updated, please re-render the item list.
+## List of Planned Metrics Events
 
-## List of Events Currently Recorded
+All events are currently implemented under the **category: lockbox-addon**. The `extra` field contains `itemid` for events pertaining to a particular Lockbox item. They are listed and grouped together below based on the contents of the event's `method` field.
 
-All events are currently implemented under the **category: lockboxV2**. The `extra` field contains `itemid` for events pertaining to a particular Lockbox item. They are listed and grouped together below based on the contents of the event's `method` field.
+1. `startup` fires when the webextension is loaded. **objects**: webextension. Note that this event fires whenever the browser is started, so is not indicative of direct user interaction with Lockbox. **value** is null.
 
-1. `startup` fires when the webextension is loaded. **objects**: webextension. Note that this event fires whenever the browser is started, so is not indicative of direct user interaction with Lockbox.
+2. `click` fires when someone clicks on one of the UI elements listed **objects**: `toolbar`, `get_mobile`, `faq`, `account_settings`, `give_feedback`, `settings_menu`, `reconnect_sync`, `signin_sync` (or whatever the menu is that contains links to account settings, faq, etc). **value** is null.
 
-2. `iconClick` fires when someone clicks the toolbar icon. **objects**: toolbar
+3. `show` events fire when various UI elements are rendered shown to the user. **objects**: `item_list_manager`, `item_list_doorhanger`, `item_detail_doorhanger` `item_detail_manager`, `new_item`, `item_edit`, `delete_confirm`, `connect_another_device` (referring to the dialog displayed after a user clicks on the button to learn about the mobile apps) **value** should be a boolean for `item_list_*` events indicating whether any logins are in the login list (e.g. `False` if the list is empty).
 
-3. `render` events fire when the item manager or doorhanger (when implemented) are rendered. **objects**: manage, doorhanger
+4. `item_add`, `item_update`, `item_delete` fire after a successful adding, editing or deleting of a credential. **objects**: manager, doorhanger (contingent on where the user initiated the action). item GUID should be in the extra field. **value** is null
```

Comment:
Item GUID might be a privacy concern, since it's unique to a specific login owned by a specific user. Would we want to hash this value before submitting?

Reply:
I'm curious about the concern.  The value itself is randomly generated (not based on or derived from the login information) from a generic entropy pool (read: not based or derived from the user).  The events are not carrying any other information about the login (e.g., username, hostname, etc) or into what the interaction is concerned (e.g., the webpage's origin).

For some of the measures in this space to have meaning, we'd have to consistently hash the GUID across platforms and instances.

Reply:
For clarification on why I was asking for this, the GUID allows me to answer, e.g., how many UNIQUE entries were edited per user. I also realized that I left this out for the credential access actions, so I added it for those as well.

If there is an alternative way, e.g. set a scalar that's like `uniqueLoginsAccessed` then we could dispense with adding the GUID.

Reply:
> If there is an alternative way, e.g. set a scalar that's like `uniqueLoginsAccessed` then we could dispense with adding the GUID.

Knowing how users are interacting with their logins has been helpful to prioritize work, so the granularity wherein is helpful if we can keep it.  I'm not sure how reliably we could track a scalar, though, especially across sessions.

Reply:
OK. To be clear, the fact that scalars naturally reset on telemetry sessions isn't a concern for me anymore.

Reply:
> I'm curious about the concern. The value itself is randomly generated (not based on or derived from the login information) from a generic entropy pool (read: not based or derived from the user). The events are not carrying any other information about the login (e.g., username, hostname, etc) or into what the interaction is concerned (e.g., the webpage's origin).

Good points. Maybe I'm being excessively cautious.